src/Mono.Android.Export/Mono.CodeGeneration/MonoAndroidExport.cs

```diff
@@ -0,0 +1,6 @@
+namespace Mono.CodeGeneration;
+
+static class MonoAndroidExport
+{
+    public const string DynamicFeatures = "Mono.Android.Export uses dynamic features.";
```

Comment:
I think I would prefer:

```
[Java.Interop.ExportAttribute] requires dynamic features.
```

Reply:
Discussed in meeting:
* We realized customers would only get a warning on `[ExportAttribute]` which says: `[RequiresUnreferencedCode ("[ExportAttribute] uses dynamic features.")]`
* The warning here should only be possibly seen by us, so it's probably fine as-is.